spec/factories/legal_aid_applications.rb

```diff
@@ -298,6 +298,32 @@
       end
     end
 
+    trait :with_multiple_proceeding_types_no_section8 do
```

Comment:
I can't see where we are using this trait?

Reply:
Good catch, I think Will added while debugging issues and then stopped using it.  
I'll remove it for now :+1: